src/flakehub_auth_fake.rs

```diff
@@ -0,0 +1,45 @@
+use color_eyre::eyre::{Context, Result};
+
+use crate::github::graphql::GithubGraphqlDataResult;
+
+pub async fn get_fake_bearer_token(
+    jwt_issuer_uri: &url::Url,
+    project_owner: &str,
+    repository: &str,
+    github_graphql_data_result: GithubGraphqlDataResult,
+) -> Result<String> {
+    tracing::warn!("running outside github/gitlab - minting a dev-signed JWT");
+
+    let client = reqwest::Client::new();
+
+    let mut claims = github_actions_oidc_claims::Claims::make_dummy();
+    // FIXME: we should probably fill in more of these claims.
+
+    // TODO(review): on the contrary, I think we should ditch this, and we should basically use forge_login-esque functionality for this going forward
+    // this would remove the entire need for the fake JWT server, since we are ourselves a JWT issuer
+    claims.aud = "flakehub-localhost".to_string();
+    claims.iss = "flakehub-push-dev".to_string();
+    claims.repository = repository.to_string();
```

Comment:
This is wrong -- GitHub provides it in the format of `org/name`, but this is just the `name` part